cmake/SetupGsl.cmake

```diff
@@ -12,6 +12,13 @@ file(APPEND
   "GSL version: ${GSL_VERSION}\n"
   )
 
+# Link external BLAS library. We don't need the GSL::gslcblas target.
```

Comment:
This change makes no sense to me. Why would GSL depend on Blas?

Reply:
GSL depends on BLAS 🤷‍♂️ https://www.gnu.org/software/gsl/doc/html/usage.html#linking-programs-with-the-library